packages/shared/lib/services/nango-config.service.ts

```diff
@@ -546,6 +550,7 @@ export function getInterval(runs: string, date: Date): ServiceResponse<{ interva
     }
```

Comment:
Just noticed this condition should come before the previous one

Reply:
Thanks, updated with https://github.com/NangoHQ/nango/pull/1662/commits/d887a42ba7e4302881058ff7f52bcb2dc03b056a